### PR TITLE
CFY-2816: refactored the checking functions

### PR DESCRIPTION
**Pull request by GlaciErrDev**
Put out the checking functions for `port` and `protocol` into util.py
and added some helpers functions for these checkings.

### CODE REVIEW

File: tests/mock/test_mock_network_plugin.py [hunk 1]
```diff
@@ -263,15 +264,15 @@ def test_check_port(self):
             check port
         """
         # port int
-        network_plugin.check_port(10)
+        utils.check_port(10)
         # port int to big
         with self.assertRaises(cfy_exc.NonRecoverableError):
-            network_plugin.check_port(65536)
+            utils.check_port(65536)
```
Comment (denismakogon) [inline tests/mock/test_mock_network_plugin.py]: utils.check_port(utils.MAX_PORT_NUMBER)

File: tests/mock/test_mock_network_plugin.py [hunk 2]
```diff
@@ -433,11 +434,11 @@ def test_check_protocol(self):
         for protocol in ["Tcp", "Udp", "Tcpudp", "Icmp", "Any"]:
```
Comment (denismakogon) [inline tests/mock/test_mock_network_plugin.py]: for protocol in utils.VALID_PROTOCOLS

File: tests/mock/test_mock_network_plugin.py (hunk 1 above)
Comment (GlaciErrDev) [inline tests/mock/test_mock_network_plugin.py]: Missed. Thanks!

File: tests/mock/test_mock_network_plugin.py (hunk 2 above)
Comment (GlaciErrDev) [inline tests/mock/test_mock_network_plugin.py]: Missed. Thanks!

File: network_plugin/utils.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        if _port_is_any(port):
+            return port.lower()
+    raise cfy_exc.NonRecoverableError(
+        "Parameter 'port' must be integer, or 'any'")
+
+def _port_is_any(source_port):
+    """
+    checks that port is 'any'
+    """
+    if source_port == 'any':
```
Comment (kostya13) [inline network_plugin/utils.py]: use     case conversions:
source_port.lower() == 'any'

**Pull request merged**